### PR TITLE
[Rule Tuning] Minor Rule Tweaks for 7.10

### PR DESCRIPTION
**Pull request by bm11100**
## Summary
Small adjustments to rule indices, names, and syntax for consistency purposes. 

`Process Injection by the Microsoft Build Engine` is still using sysmon CreateRemoteThread in the syntax.

Do we want to edit any additional rules (packetbeat-*, etc) to add the endpoint index?

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by brokensound77** [comment]
These changes lgtm so far. Does winlogbeat populate `event.type` for `CreateRemoteThread`? As far as adding endpoint to packetbeat rules, we'd definitely have to go through them individually to determine if they would work based on the available fields

**Review by brokensound77 — APPROVED** [review]
Other than the question on `CreateRemoteThread`, LGTM 👍

File: tests/test_all_rules.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def test_required_tags(self):
             'endgame-*': {'all': ['Endpoint']},
             'logs-aws*': {'all': ['AWS']},
             'logs-endpoint.alerts-*': {'all': ['Endpoint']},
-            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux']},
+            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux', 'Endpoint']},
```
Comment (brokensound77) [inline tests/test_all_rules.py:235]:
```suggestion
            'logs-endpoint.events.*': {'any': ['Windows', 'macOS', 'Linux', 'Host']},
```

Use `Host` for the rule tags instead

File: detection_rules/packaging.py [hunk 2]
```diff
@@ -318,7 +318,7 @@ def get_summary_rule_info(r: Rule):
             return rule_str
 
         def get_markdown_rule_info(r: Rule, sd):
-            rules_dir_link = f'https://github.com/elastic/detection-rules/tree/v{self.name}/rules/{sd}/'
+            rules_dir_link = f'https://github.com/elastic/detection-rules/tree/{self.name}/rules/{sd}/'
```
Comment (brokensound77) [inline detection_rules/packaging.py:321]: we don't include `v` in branch names

Comment (rw-access) [reply detection_rules/packaging.py:321]: tree works for tags. so i think this is right?
tags have `major.minor.patch` vs release branches which are `major.minor`

Comment (brokensound77) [reply detection_rules/packaging.py:321]: oh so maybe that is why I did it - to tie it to the tag and not the release branch. I will revert this then - thanks!

**Review by rw-access — APPROVED** [review]

File: detection_rules/packaging.py (hunk 2 above)
Comment (rw-access) [reply detection_rules/packaging.py]: sweet. also `sd` is probably not the most descriptive name for a var.
we can come back to this and add an inline comment for the change

Comment (brokensound77) [reply detection_rules/packaging.py]: 🙂

**Pull request merged**